Feature/Room.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Validating Rooms are displayed when date is input
+
+  Scenario Outline: The rooms are visible when user inputs date
+    Given The user is on the home page
+    When The user clicks on the "Rooms" top menu
+    And The user clicks on the *Click to Select Dates* date picker
```

Comment:
add a validation step to check that you are on the Rooms page

Reply:
Done